rpc/rpc-wrappers.go

```diff
@@ -372,6 +374,23 @@ func NewRegistrationAuthorityServer(rpc Server, impl core.RegistrationAuthority,
 		return
 	})
 
+	rpc.Handle(MethodDeactivateAuthorization, func(ctx context.Context, req []byte) (response []byte, err error) {
+		var authz core.Authorization
+		err = json.Unmarshal(req, &authz)
+		if err != nil {
+			// AUDIT[ Error Conditions ] 9cc4d537-8534-4970-8665-4b382abe82f3
```

Comment:
Yeah, the `AUDIT[ Error Conditions ]` comments were intended for hypothetical tooling we thought we needed, but turns out we don't need it.